test/achievement-specs.js

```diff
@@ -74,25 +114,74 @@ describe('achievement', () => {
   })
 
   context('POST /achievement', () => {
-    it('should create achievement with sequence=sequence, name=name, and distance=distance', async () => {
+    it('should create achievement with sequence=sequence, name=name, distance=distance,' +
+      'description=description, icon_name=icon_name, map_image=map_image,' +
```

Comment:
Add a space so the concatenated string is readable
```suggestion
      ' description=description, icon_name=icon_name, map_image=map_image,' +
```